default.nix

```diff
@@ -62,6 +63,44 @@ with holonix.pkgs;
     pkgs = holonix.pkgs;
    }).buildInputs
 
+   ++ ([(
+    holonix.pkgs.writeShellScriptBin "hc-install" ''
+     cargo install --path crates/holochain
+     echo 'holochain installed!'
+     echo
+
+     hc-doctor
+    ''
+   )])
+
+   ++ ([(
+    holonix.pkgs.writeShellScriptBin "hc-uninstall" ''
+     rm $CARGO_INSTALL_ROOT/bin/holochain
+     echo 'holochain uninstalled!'
+     echo
+
+     hc-doctor
+    ''
+   )])
```

Comment:
@thedavidmeister people need to build `dna-util` as well. And maybe lair? So, should these be namespaced, `hc-install-holochain` etc?